docker/buildkite/docker-compose-local.yml

```diff
@@ -86,8 +91,9 @@ services:
     build:
       context: ../../
       dockerfile: ./docker/buildkite/Dockerfile
-    command: make cover_profile
+    command: sh -c "make .just-build && make install-schema && make cover_profile"
```

Comment:
poking at this locally, I kinda think we don't want `make install-schema` here.  it fails if the schema already exists, which means you can't re-run unit tests without tossing and restarting the cassandra container :\

Reply:
This is not the tool that you use to run unit tests. This is a "close to CI" way to simulate what is running on buildkite, so I wanted it to behave as CI run. If it will be needed, we can separate it to an extra step, but it was useful to debug the problem

Reply:
I suppose in that scenario, people would have to drop cassandra each time?  and that's reasonable because it matches CI?

which may be reasonable, this is in a buildkite folder after all.  it's not a totally general thing.  just checking / understanding.